src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,29 @@
+package tests;
+
+import driver.DriverManager;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Listeners;
+import org.testng.asserts.SoftAssert;
+import util.TestListener;
+
+import java.util.logging.Logger;
+
+@Listeners({TestListener.class})
+public class BaseTest {
+
+    protected WebDriver driver;
+    protected Logger logger = Logger.getGlobal();
```

Comment:
please use one logger type in your project - log4j

Reply:
Refactored, thank you.
Please see the changes in the following commit.